<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>788</width>
    <height>597</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>301</width>
     <height>581</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QToolBox" name="toolBox_QCoreApplication">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>281</width>
      <height>531</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>6</number>
    </property>
    <widget class="QWidget" name="widget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>281</width>
       <height>293</height>
      </rect>
     </property>
     <attribute name="label">
      <string>QCoreApplication</string>
     </attribute>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>271</width>
        <height>301</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="pushButton_Dirpath">
         <property name="text">
          <string>applicationDirpath（）</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_FilePath">
         <property name="text">
          <string>applicationFilePath()</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_appName">
         <property name="text">
          <string>applicationName()</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_setappName">
         <property name="text">
          <string>setApplicationName()</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_librayPaths">
         <property name="text">
          <string>librayPaths()</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_organizationName">
         <property name="text">
          <string>organizationName()</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_exit">
         <property name="text">
          <string>exit()</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>281</width>
       <height>293</height>
      </rect>
     </property>
     <attribute name="label">
      <string>QFile</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>281</width>
        <height>141</height>
       </rect>
      </property>
      <property name="title">
       <string>静态函数</string>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>261</width>
         <height>111</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QPushButton" name="pushButton_copys">
          <property name="text">
           <string>copy()</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="pushButton_existss">
          <property name="text">
           <string>exists()</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="pushButton_removes">
          <property name="text">
           <string>remove()</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="pushButton_renames">
          <property name="text">
           <string>rename()</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="pushButton_moveToTrashs">
          <property name="text">
           <string>moveToTrash</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_5">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>150</y>
        <width>281</width>
        <height>161</height>
       </rect>
      </property>
      <property name="title">
       <string>成员函数</string>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>261</width>
         <height>111</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QPushButton" name="pushButton_copy">
          <property name="text">
           <string>copy()</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="pushButton_exists">
          <property name="text">
           <string>exists()</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="pushButton_remove">
          <property name="text">
           <string>remove()</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="pushButton_rename">
          <property name="text">
           <string>rename()</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="pushButton_moveToTrash">
          <property name="text">
           <string>moveToTrash</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>281</width>
       <height>293</height>
      </rect>
     </property>
     <attribute name="label">
      <string>QFileinfo</string>
     </attribute>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>281</width>
        <height>328</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QPushButton" name="pushButton_absoluteFilepath">
           <property name="text">
            <string>absoluteFilePath</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_absolutePath">
           <property name="text">
            <string>absolutePath</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QPushButton" name="pushButton_fileName">
           <property name="text">
            <string>fileName</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_filepath">
           <property name="text">
            <string>filepath</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QPushButton" name="pushButton_size">
           <property name="text">
            <string>size</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_path">
           <property name="text">
            <string>path</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="3" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QPushButton" name="pushButton_baseName">
           <property name="text">
            <string>baseName</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_completeBaseName">
           <property name="text">
            <string>completeBaseName</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="4" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <widget class="QPushButton" name="pushButton_suffix">
           <property name="text">
            <string>suffix</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_completeBaseName_2">
           <property name="text">
            <string>completeSuffix</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="5" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_10">
         <item>
          <widget class="QPushButton" name="pushButton_isDir">
           <property name="text">
            <string>isDir</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_isFile">
           <property name="text">
            <string>isFile</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="6" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_11">
         <item>
          <widget class="QPushButton" name="pushButton_isExecutable">
           <property name="text">
            <string>isExecutable</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_birthTime">
           <property name="text">
            <string>birthTime</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="7" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_12">
         <item>
          <widget class="QPushButton" name="pushButton_lastModified">
           <property name="text">
            <string>lastModified</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_lastRead">
           <property name="text">
            <string>lastRead</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="8" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_13">
         <item>
          <widget class="QPushButton" name="pushButton_exists_s">
           <property name="text">
            <string>静态函数exists</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_exists_2">
           <property name="text">
            <string>exists</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="page">
     <attribute name="label">
      <string>QDir</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_6">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>281</width>
        <height>131</height>
       </rect>
      </property>
      <property name="title">
       <string>静态函数</string>
      </property>
      <widget class="QWidget" name="">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>271</width>
         <height>100</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QPushButton" name="pushButton_tempPath">
          <property name="text">
           <string>tempPath</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="pushButton_rootPath">
          <property name="text">
           <string>rootPath</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="pushButton_homepath">
          <property name="text">
           <string>homepath</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="pushButton_drives">
          <property name="text">
           <string>drives</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="pushButton_currentPath">
          <property name="text">
           <string>currnetPath</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="pushButton_setCurrent">
          <property name="text">
           <string>setCurrent</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_7">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>130</y>
        <width>281</width>
        <height>161</height>
       </rect>
      </property>
      <property name="title">
       <string>文件或目录操作</string>
      </property>
      <widget class="QWidget" name="">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>261</width>
         <height>131</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0">
         <widget class="QPushButton" name="pushButton_mkdir">
          <property name="text">
           <string>mkdir</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="pushButton_rmdir">
          <property name="text">
           <string>rmdir</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="pushButton_remove_2">
          <property name="text">
           <string>remove</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="pushButton_rename_2">
          <property name="text">
           <string>rename</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="pushButton_SetPath">
          <property name="text">
           <string>SetPath</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="pushButton_removeRecursively">
          <property name="text">
           <string>removeRecursively</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page_4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>281</width>
       <height>293</height>
      </rect>
     </property>
     <attribute name="label">
      <string>QTemporayDir</string>
     </attribute>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>271</width>
        <height>271</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="pushButton_temptemp">
         <property name="text">
          <string>1.在系统临时目录下创建临时文件夹</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_diytemp">
         <property name="text">
          <string>2.在指定目录下创建临时文件夹</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_currenttemp">
         <property name="text">
          <string>3.在当前目录下创建临时文件夹</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="page_5">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>281</width>
       <height>293</height>
      </rect>
     </property>
     <attribute name="label">
      <string>QTempporaryFile</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="page_6">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>281</width>
       <height>293</height>
      </rect>
     </property>
     <attribute name="label">
      <string>QFileSystemWatcher</string>
     </attribute>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>281</width>
        <height>261</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QPushButton" name="pushButton_add">
         <property name="text">
          <string>addPath()并开始监视</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_remove_3">
         <property name="text">
          <string>removePath()并停止监视</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_direct">
         <property name="text">
          <string>directories()</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_files">
         <property name="text">
          <string>files()</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>10</y>
     <width>461</width>
     <height>581</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>421</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>391</width>
       <height>30</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="pushButton_openfile">
        <property name="text">
         <string>打开文件</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_openDir">
        <property name="text">
         <string>打开目录</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_cleartext">
        <property name="text">
         <string>清除文本框</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>421</width>
      <height>27</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>目录</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_dirpath"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>421</width>
      <height>27</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>文件</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_filepath"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>421</width>
      <height>411</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>信息</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="plainTextEdit">
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
